keyboards/ms_sculpt/rules.mk

```diff
@@ -0,0 +1,13 @@
+# This file intentionally left blank
+DEBOUNCE_TYPE = sym_eager_pk
+DEBOUNCE = 3
+
+RGBLIGHT_ENABLE         = yes       # Enable global lighting effects. Do not enable with RGB Matrix
+RGBLIGHT_ANIMATIONS     = yes
+RGB_MATRIX_ENABLE       = no        # Enable per-key coordinate based RGB effects. Do not enable with RGBlight
+
+RGBLIGHT_FULL_POWER     = yes
+
+RAW_ENABLE = yes
+
```

Comment:
the rgblight values aren't used and should be removed.  The rest don't appear to be used. 
```suggestion
```